package.json

```diff
@@ -20,7 +20,9 @@
         "markdownlint": "0.31.1"
     },
     "devDependencies": {
-        "eslint": "^9.9.1"
+        "eslint": "^9.9.1",
+        "eslint-plugin-jest": "^28.11.0",
+        "eslint-plugin-mocha": "^10.5.0"
```

Comment:
need to be added to `dependencies` (and/or `peerDependencies`?) for the plugins to be automatically added to the "top-level" node_modules, and thus available when running eslint...

Reply:
as we don't need both, they should go in `peerDependencies` and eslint should go there too

Reply:
if we don't add it to `dependencies`, these plugins will not be included when depending on this plugin (in the "user" package, like cloudserver/...), as peer dependencies do not get installed (just display a warning) : and thus will have more dependencies to include in every package...

The `peerDependencies` is not really needed here, what I really want is a dependency : and peerDependency may be added simply to avoid having duplicating versions of the dependency...

Reply:
But that's the point that we don't want to automatically add it by default, it should be the source repo that picks if it wants to install the plugin jest or mocha based on what it needs to avoid unnecessary dependency.

But maybe they are small enough to ignore an extra unused dependency.

Reply:
I would rather it "just works", and the dependencies are added automatically ; but indeed it may pull quite a few deps, esp. the JEST plugin. Maybe we actually already have everything, so not much difference; but I am not sure... 

The only "safeguard" of sorts is that this is this whole package is a **_dev_** dependency, so will not have any effect on prod.

That said, if we do not have the dependencies, the user (dev) experience will not be very good : unless the appropriate plugin is explicitely specified, then eslint will most likely fail to run with a cryptic message (or worse, the extra check will silently not be performed)